client/js/lounge.js

```diff
@@ -1331,7 +1331,7 @@ $(function() {
 					} else {
 						var current = $(this);
 
-						if (current.attr("class") === "date-marker") {
+						if (current.hasClass("date-marker") && prev.hasClass("date-marker")) {
```

Comment:
So, if we have:

* unread-marker
* date-marker
* date-marker
* message

We want to end up with 

* unread-marker
* date-marker
* message

Correct?

I'm pretty sure this code will not do that, because it will set the first to be unread-marker, then it will check that both 0 and 1 are date-markers, but they aren't. So it will return false. This will then stop it doing the job, I think.

Am I missing something?